api/v1alpha1/groupversion_info.go

```diff
@@ -26,7 +26,7 @@ import (
 
 var (
 	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{Group: "app.example.com", Version: "v1alpha1"}
+	GroupVersion = schema.GroupVersion{Group: "remediation.medik8s.io", Version: "v1alpha1"}
```

Comment:
sounds a bit generic to me. I would expect any remediation plugin to have its own group.